### DISCUSSION TITLE
How can I change the size of the free text annotation box?

### DISCUSSION THREAD

**Discussion by wj-qiao** [sumatrapdfreader/sumatrapdf#3753] [Q&A]
Hi there,

I'm using the latest pre-release version of SumatraPDF,  and I insert a free text annotation, while the annotation box is too large, so how can I change the size of the free text annotation box to resize the distribution of text?

**Comment by kjk** [comment]
Currently it's not possible.

**Reply by wj-qiao** [reply]
Thus， will this feature be added in the future version?

**Reply by kjk** [reply]
quite possibly

**Reply by gwebbgit** [reply]
Seems like the easiest fix might be to allow pressing+holding Ctrl to allow resizing of a currently selected Free Text annotation box by a cursor drag.  Currently the cursor drag only repositions the box.

Or instead of press+hold Ctrl, the cursor could resize the box if dragged from close to the box's perimeter;  otherwise, dragging the box from closer to its center would reposition the box as usual.

**Comment by pmcstjean** [comment]
If it's too large I guess you can add line ending manually to "wrap" the text. My problem is that it's too small.

**Comment by andrechalella** [comment]
Would it be too difficult to allow editing `dx` and `dy` in the dialog?

**Reply by GitHubRulesOK** [reply]
@andrechalella
 I had opened that issue some time ago and others have tried some attempts to allow "handles" so see https://github.com/sumatrapdfreader/sumatrapdf/issues/1665#issuecomment-657051120 and collective https://github.com/sumatrapdfreader/sumatrapdf/issues/2485#issuecomment-1099242730 and note https://github.com/sumatrapdfreader/sumatrapdf/pull/5196 whiich is a more recent "proposal to add some adjustments"
